source/utils/paragraphHelper.py

```diff
@@ -136,6 +126,15 @@ def speakParagraph(ti: textInfos.TextInfo) -> None:
 		speech.speakMessage(_("blank"))
 
 
+def _notFoundMessage(nextParagraph: bool):
+	if nextParagraph:
+		# Translators: this message is given when there is no next paragraph
```

Comment:
adding some more context may be helpful here

```suggestion
		# Translators: this message is given when there is no next paragraph when navigating by paragraph
```